api/proto/v1/proto_broadage.proto

```diff
@@ -56,6 +56,8 @@ message MatchKey {
   string competition_fid = 1;
   /// fid is the unique id of the match.
   string fid = 2;
+  /// project_key is the unique key of the project.
```

Comment:
En serio os lo digo, para decir estas chorradas, que encima son mentira, no le pidáis a copilot que ponga comentarios.